config/vufind/markdown.ini

```diff
@@ -30,3 +30,39 @@
 ; blocks are too deeply-nested.
 ;max_nesting_level = 10
 
+; Which extension you want to activate. List of extension names separated by comma. Available extensions as of league/commonmark
+; version 1.4: ExternalLink, HeadingPermalink, SmartPunct, TableOfContents
+; Some of them could have a configuration, see sections bellow
+;extensions = ExternalLink,HeadingPermalink,SmartPunct,TableOfContents
+
+; See https://commonmark.thephpleague.com/1.4/extensions/external-links/
+[ExternalLink]
+; This should be always set, if you wan't to use this extension. You can use regular
```

Comment:
Small typo: `wan't` for `want`. I'd fix this myself, but I don't seem to have edit permission.

Reply:
Fixed